### DISCUSSION TITLE
How to bind download for specific path?

### DISCUSSION THREAD

**Discussion by VolkovIlia** [qutebrowser/qutebrowser#6641] [Q&A] [answered]
Hi!

How to bind keys in config.py to download hinted link-file to specific directory. For example I wont to bind PDF to /home/user/PDF, image to /home/user/images etc...

**Comment by toofar** [comment, marked as answer]
You could add a binding that sets the download directory then initiates hinting, you would have to remember to set the download directory back afterwards though because the hint command isn't blocking

**Reply by VolkovIlia** [reply]
Found the solution `config.bind(',db', "set downloads.location.directory /home/user/path ;; hint links download")`
